cmake/linux/package_linux.sh.in

```diff
@@ -6,11 +6,10 @@
 # Notes:    Will attempt to fetch linuxdeployqt automatically (x86_64 only)
```

Comment:
The comment sauys `x86_64` only, is this statement still true?

Reply:
According to https://github.com/probonopd/linuxdeployqt/releases, it's still true. To build AppImages for ARM, one should build `linuxdeployqt` from source.